src/layers/legacy/medCoreLegacy/database/medDatabaseImporter.cpp

```diff
@@ -357,3 +357,20 @@ QString medDatabaseImporter::ensureUniqueSeriesName(const QString seriesName, co
 
     return newSeriesName;
 }
+
+void medDatabaseImporter::createJsonMetadataEntry(medAbstractData *medData, int seriesDbId)
```

Comment:
check if column exist
https://stackoverflow.com/questions/18920136/check-if-a-column-exists-in-sqlite